### ISSUE TITLE
HI! is there a pretrained model ?

### ISSUE THREAD

**Issue by wqz960** [lippman1125/pytorch_FAN#2]
Thank you for your code, do you test it, can you provide me the pretrained model?

**Comment by wqz960** [comment]
or can you tell me the training loss in different stage, i just run it and the loss seems to be very low , about 0.03 in the beginning, and drop rapidly to 0.005, is there some bug? i trained on 300WLP. Looking forward for your kind respond!

**Comment by xsacha** [comment]
Loss values in general don't mean much as long as they go down.

You will also notice accuracy on validation hits 100% in a few epochs.

Probably needs better test set really. I've no idea which epoch is best.

**Comment by EdwardChCriollo** [comment]
I have the same doubt, training loss drop to 0.004 and so on, but mi training accuracy stay in 0.0000 on 700 epoch of 8971, what is the reason to obtain these values?, is there any problem?, thanks in advance...

**Comment by sxs4337** [comment]
Ping! Any possibility of a pre-trained model?

**Comment by AmitSharma1127** [comment]
A pre-trained model would be of huge help
